install.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root."
+   exit 1
+fi
+
+PREFIX=/usr
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+# Install JobAdder source code + binaries
+# FIXME: how to make setup.py ignore test folder??
+cd $SCRIPTPATH/src/
+mv ./test/__init__.py ./test/init
+./setup.py install --prefix=$PREFIX
+mv ./test/init ./test/__init__.py
+cd $SCRIPTPATH
+
+# TODO: when remote is ready, set SUID
+
+# Install Systemd unit files
+SERVER_SRV_FILE=/etc/systemd/system/JobAdderServer.service
+
+cp ./data/bin/JobAdderServer.service $SERVER_SRV_FILE
```

Comment:
I do not think we need a symbolic link.

Reply:
I'm asking because the service files on my system are almost all symbolic links; I thought there may be a reason for that.

Reply:
Actually, now that I think about it we should definitely copy the service file instead of linking it.
Otherwise the service would be probably be linked to a file in the user space.